src/main/java/org/openrewrite/openapi/swagger/MigrateSwaggerDefinitionToOpenAPIDefinition.java

```diff
@@ -0,0 +1,114 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.openapi.swagger;
+
+import org.jspecify.annotations.Nullable;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.AnnotationMatcher;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class MigrateSwaggerDefinitionToOpenAPIDefinition extends Recipe {
+
+    private static final String FQN_SWAGGER_DEFINITION = "io.swagger.annotations.SwaggerDefinition";
+    private static final String FQN_OPENAPI_DEFINITION = "io.swagger.v3.oas.annotations.OpenAPIDefinition";
+    private static final String FQN_SERVER = "io.swagger.v3.oas.annotations.servers.Server";
+
+    @Override
+    public String getDisplayName() {
+        return "Migrate from `@SwaggerDefinition` to `@OpenAPIDefinition`";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Migrate from `@SwaggerDefinition` to `@OpenAPIDefinition`.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return Preconditions.check(
+                new UsesType<>(FQN_SWAGGER_DEFINITION, false),
+                new JavaIsoVisitor<ExecutionContext>() {
+                    private final AnnotationMatcher annotationMatcher = new AnnotationMatcher(FQN_SWAGGER_DEFINITION);
+
+                    @Override
+                    public J.@Nullable Annotation visitAnnotation(J.Annotation annotation, ExecutionContext ctx) {
+                        J.Annotation ann = super.visitAnnotation(annotation, ctx);
+
+                        if (annotationMatcher.matches(ann)) {
+                            Map<String, Expression> args = AnnotationUtils.extractAnnotationArgumentAssignments(ann);
+
+                            StringBuilder tpl = new StringBuilder("@OpenAPIDefinition(\n");
+                            List<Expression> tplArgs = new ArrayList<>();
+                            List<String> parts = new ArrayList<>();
+
+                            Expression basePath = args.get("basePath");
+                            Expression host = args.get("host");
+                            Expression schemes = args.get("schemes");
+                            String servers = "";
+                            if (basePath != null && host != null && schemes != null) {
+                                tpl.append("servers = {\n");
+                                for (Expression scheme : ((J.NewArray) schemes).getInitializer()) {
```

Comment:
If folks use `schemes = SwaggerDefinition.Scheme.HTTPS` then this throws a class cast exception:
```
Caused by: java.lang.ClassCastException: class org.openrewrite.java.tree.J$FieldAccess cannot be cast to class org.openrewrite.java.tree.J$NewArray (org.openrewrite.java.tree.J$FieldAccess and org.openrewrite.java.tree.J$NewArray are in unnamed module of loader 'app')
	at org.openrewrite.openapi.swagger.MigrateSwaggerDefinitionToOpenAPIDefinition$1.visitAnnotation(MigrateSwaggerDefinitionToOpenAPIDefinition.java:75)
	at org.openrewrite.openapi.swagger.MigrateSwaggerDefinitionToOpenAPIDefinition$1.visitAnnotation(MigrateSwaggerDefinitionToOpenAPIDefinition.java:55)
	at org.openrewrite.java.tree.J$Annotation.acceptJava(J.java:233)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:250)
```
I think that's common enough to warrant proper handling here.